x-pack/plugins/ml/public/application/components/annotations/annotation_flyout/index.tsx

```diff
@@ -85,7 +86,8 @@ export class AnnotationFlyoutUI extends Component<CommonProps & Props> {
   public annotationSub: Rx.Subscription | null = null;
 
   componentDidMount() {
-    this.annotationSub = annotation$.subscribe((v) => {
+    const { annotationUpdatesService } = this.props;
+    this.annotationSub = annotationUpdatesService.get().subscribe((v) => {
```

Comment:
You should use `updates$` instead

Reply:
Using `update$` here will make the text input area in the annotation flyout not update when typing.

Reply:
Yes, you should have another method. So 
-  existing `updates$` can be renamed to `isAnnotationInitialized$` or something like that
- new `updates$` methods should be just 
```typescript
public updates$() {
    return this._annotation$.asObservable();
  }
```

Reply:
Updated here https://github.com/elastic/kibana/pull/83216/commits/0cf7ad04d7cb582f445be2eca121a0f5946ab03e